handbook/communication/rfcs/index.md

```diff
@@ -1,139 +1,173 @@
+
+## RFC formats we’ve found helpful in the past
+
+The default RFC template is minimal and contains just the attributes you need to get started on a new RFC. However, we’ve found a few different structured formats and workflows to be helpful in achieving specific outcomes with RFCs.
+
+Instead of providing these formats in the default template, we document the RFC formats and workflows we’ve found helpful in the handbook. This way, we keep RFCs lightweight and flexible, but can also keep iterating and improving on formats that work well for specific needs.
+
+If you find that a new format works well for achieving a distinct goal and could be of use in the future, please add it to our list of formats.
+
+### The default format
+
+The default format has just the required attributes and can be used as the starting point for any RFC.
+
+[Get the template](https://docs.google.com/document/d/1VV0ddLmMrcU2IWo_s4xm8Q8UtGnreyuhIRPJY0Dh5NI/edit?usp=sharing) →
+
+### Goal: Framing problems, proposing solutions, and making decisions
+
+This format is helpful for framing a problem, proposing solutions (or collecting feedback so as to propose a solution), and for ultimately [making decisions](../decisions.md).
+
+This is our typical format for defining what we feel is important to work on, and acts as a constant touchpoint throughout our work from conceptualization to implementation and measuring success.
+
+In this format, we find it helpful to include:
+
+- **Tags** for the people involved and their roles in the decision-making framework (including the [decider](../decisions.md#decider), input providers, approvers, and [approvals](../decisions.md#approvers)).
+- **Tags** for any teams that will be affected either directly or indirectly, and should have the opportunity to review and contribute to the RFC.
+- **Background**: Just enough context necessary to frame the rest of the RFC. The content should be indisputable facts, not opinion.
+- **Problem**: A description of the problem that this RFC is trying to address, the constraints, and why this problem is worth solving now.
+- **Proposed solution** (optional): A description of how to solve the problem. It’s okay to skip this section if you just want to define a problem. Otherwise, it can be helpful for thinking of solutions or to hand off ownership to someone else.
+- **Definition of success**: How do we know if this proposal was successful? Are there any metrics we need to start tracking?
+
+[Get the template](https://docs.google.com/document/d/1FJ6AhHmVInSE22EHcDZnzvvAd9KfwOkKvFpx7e346z4/edit?usp=sharing) →
+
+### Goal: Surfacing a Tension
+
+This format is helpful for surfacing a Tension. A Tension is somewhere between a problem, a feeling (of unease), and a person’s (unmet) needs. When Tensions come up, they describe a sense there’s a gap between the current reality and an ideal potential future.
```

Comment:
It feels weird/inconsistent to capitalize "Tension" here. For example, the previous section isn't "This format is helpful for framing a Problem."

```suggestion
This format is helpful for surfacing a tension. A tension is somewhere between a problem, a feeling (of unease), and a person’s (unmet) needs. When tensions come up, they describe a sense there’s a gap between the current reality and an ideal potential future.
```